web/router.ex

```diff
@@ -17,6 +17,7 @@ defmodule Notox.Router do
     pipe_through :browser # Use the default browser stack
 
     get "/", PageController, :index
+    get "/notes", NoteController, :index
```

Comment:
If I was reasonable certain I was going to add additional routes to this resource, I'd probably do:

```
resources "/notes", NoteController, only: [:index]
```

Then you only have to add options to the `only` when new routes come along.